README.md

```diff
@@ -0,0 +1,14 @@
+[![Build](https://github.com/gentoo-ev/images.gentoo-ev.org/actions/workflows/build.yml/badge.svg)](https://github.com/gentoo-ev/images.gentoo-ev.org/actions/workflows/build.yml)
+
+
+# How to Build and Run (on Gentoo)
+
+```console
+# sudo emerge app-emulation/docker app-emulation/docker-compose
+# sudo /etc/init.d/docker start
+# docker network create --internal ssl-reverse-proxy
+# docker-compose up -d --build
+# chromium http://127.0.0.1:50380/ &
```

Comment:
Any reason why this must be Chromium? :)
```suggestion
# ${BROWSER} http://127.0.0.1:50380/ &
```

Reply:
Hi @ulm, it does not have to be Chromium, and I am using Firefox more than Chromium personally in a mix. You're seeing this at least the third time though — both https://github.com/gentoo-ev/www.gentoo.de and https://github.com/gentoo-ev/www.gentoo-ev.org already have exactly this. I'll keep it as is, please feel free to adjust it to Firefox or `"${BROWSER}"` or `"${BROWSER}"` for all three after if you think it matters. If Chromium is more evil than I am aware it is, that would be cool hear more about it off-GitHub. Thanks!